cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -170,7 +172,22 @@ export class Analysis extends Application {
 
     protected selectTarget(target: string[]): void {
         for (let i = 0; i < target.length; i++) {
-            cy.get("div.pf-v5-c-empty-state__content").children("h4").contains(target[i]).click();
+            if (
+                target.includes("OpenJDK 11") ||
+                target.includes("OpenJDK 17") ||
+                target.includes("OpenJDK 21")
+            ) {
+                click(openjdkToggleButton);
+                clickByText(dropDownMenu, target[i]);
+            } else if (target.includes("camel:3") || target.includes("camel:4")) {
+                click(camelToggleButton);
+                clickByText(dropDownMenu, target[i]);
+            } else {
+                cy.get("div.pf-v5-c-empty-state__content")
+                    .children("h4")
+                    .contains(target[i])
+                    .click();
+            }
```

Comment:
Hi Karishma .. See if this can work for all targets .

```protected selectTarget(target: string[]): void {
        for (let i = 0; i < target.length; i++) {
            let target1 = target[i].split(" ");
            cy.get("div.pf-v5-c-empty-state__content").children("h4").contains(target1[0]).click();
            if(cy.get(`#${target1[0]}-toggle`)){
                cy.get(`#${target1[0]}-toggle`).click();
                clickByText(dropDownMenu, target[i]);
            }
        }```

Reply:
Thanks Shveta, tried this method but doesn't work for all targets. Also tried to modified according to all targets but success. So the previous one works, i have modified it accordingly.